ratenyu/courses/templates/courses/detail.html

```diff
@@ -38,13 +38,14 @@ <h2>{{ reviews_list|length }} Reviews</h2>
     <ul class="detail-side-links">
         <li class="detail-side-links"><span class="rev-title">Professor:</span></li>
         {% for professor in professors_list %}
-        <li class="detail-side-links"><a href="">{{ professor.name }}</a></li>
+            <li class="detail-side-links"><a href='#' onclick = "professorSelect('{{professor.name}}')">{{ professor.name }}</a></li>
         {% endfor %}
+        <li class="detail-side-links"><a href='#' onclick = "professorSelect('{{professors_list.name}}')">All</a></li>
     </ul>
 
     <div class="all-reviews">
         {% for review in reviews_list %}
-        <div class="review">
+        <div class="review" review_professor = "{{review.professor_obj.name}}">
```

Comment:
I feel like you don't need this. Can we use the professor name that is mentioned in detail-side-links class tag.